coccinelle/run-coccinelle.sh

```diff
@@ -15,7 +15,8 @@ for SCRIPT in ${@-$top/coccinelle/*.cocci} ; do
         echo "--x-- Processing $SCRIPT --x--"
         TMPFILE=`mktemp`
         echo "+ spatch --sp-file $SCRIPT $args ..."
-        spatch --sp-file $SCRIPT $args $files 2>"$TMPFILE" || cat "$TMPFILE"
-        rm "$TMPFILE"
+        parallel --halt now,fail=1 --keep-order --noswap --max-args=20 \
```

Comment:
i have never heard of parallel before. maybe say "GNU parallel" or so, to clarify that this is some seperate software project.

also, maybe make the script fail early with a friendly error message if gnu parallel is not installed?